src/storage/hummock_test/Cargo.toml

```diff
@@ -9,6 +9,7 @@ repository = { workspace = true }
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+async-stream = "0.3.3"
```

Comment:
Use `async-stream = "0.3"` to make it consistant with the specification in other packages.